source/broadband/webpa_table.c

```diff
@@ -789,14 +799,20 @@ static void getWritableParams(char *paramName, char ***writableParams, int *para
 					}
 					else
 					{
-						(*writableParams)[cnt1] = (char *)malloc(sizeof(char) * MAX_PARAMETERNAME_LEN);
-						strcpy((*writableParams)[cnt1],tempStr);	
+						paramList[cnt1] = strdup(tempStr);	
 						cnt1++;
 					}	
 				}
 			}
 			*paramCount = writableCount = cnt1;
 			WalPrint("writableCount %d\n",writableCount);
```

Comment:
Change this log to Infy level so we capture cases where ret is success but writable count is 0